lib/SILOptimizer/Transforms/SimplifyCFG.cpp

```diff
@@ -3190,6 +3190,7 @@ bool simplifyToSelectValue(SILBasicBlock *MergeBlock, unsigned ArgNum,
   }
   
   SmallVector<std::pair<SILValue, SILValue>, 8> Cases;
+  llvm::SmallSetVector<SILValue, 8> CaseLiterals;
```

Comment:
Using SmallDenseMap in new version - need to keep the result to check if its value matches